gdal/port/cpl_swift.cpp

```diff
@@ -170,6 +184,258 @@ bool VSISwiftHandleHelper::GetConfiguration(CPLString& osStorageURL,
     return true;
 }
 
+/************************************************************************/
+/*                      CreateAuthV3RequestObject()                     */
+/************************************************************************/
+
+CPLJSONObject VSISwiftHandleHelper::CreateAuthV3RequestObject()
+{
+    CPLString osUser = CPLGetConfigOption("SWIFT_USER", "");
+    CPLString osKey = CPLGetConfigOption("SWIFT_KEY", "");
+
+    CPLJSONObject user;
+    user.Add("name", osUser);
+    user.Add("password", osKey);
```

Comment:
SWIFT_KEY used as password. Will need to be documented. But seeing that method, I'm wondering about the value of all those new config options. Wouldn't provided directly the serialized JSon document in a config option be more convenient ?

Reply:
All those config options are the same as those of the python-swiftclient except they use `SWIFT_` prefix instead of `OS_`:
https://docs.openstack.org/python-swiftclient/latest/cli/index.html#authentication

So I would stick with that.

Or maybe support directly those options with `OS_` prefix. They are used for authentication across the whole OpenStack environment:
https://docs.openstack.org/python-openstackclient/train/cli/authentication.html

Reply:
> Or maybe support directly those options with `OS_` prefix

That would seem the best option to me.

Reply:
@talaj Will you change to options using the OS_ prefix ?

Reply:
Yes, I will! I just haven’t had time to make it yet.